android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/ext/Calendar.kt

```diff
@@ -60,6 +67,6 @@ internal fun Calendar.minMonth(): Int = getMinimum(Calendar.MONTH)
 internal fun Calendar.maxMonth(): Int = getActualMaximum(Calendar.MONTH)
 internal fun Calendar.minDay(): Int = getMinimum(Calendar.DAY_OF_MONTH)
 internal fun Calendar.maxDay(): Int = getActualMaximum(Calendar.DAY_OF_MONTH)
-internal fun Calendar.minYear(): Int = getMinimum(Calendar.YEAR)
-internal fun Calendar.maxYear(): Int = getActualMaximum(Calendar.YEAR)
+internal fun Calendar.minYear(): Int = DEFAULT_MIN_YEAR
+internal fun Calendar.maxYear(): Int = DEFAULT_MAX_YEAR
```

Comment:
I think we should leave these as they are. Different types of `Calendar`s may be able to specify different minimums and maximums. Code for time is notoriously difficult to get correct, so in this case I think we should rely on the standard.